File: 3515-find-if-digit-game-can-be-won/find-if-digit-game-can-be-won.java
class Solution {
    public boolean canAliceWin(int[] nums) {
        int dd = 0;
        int sd = 0;
        for(int i=0;i<nums.length;i++)
        {
            if(nums[i]>9)
            {
                dd += nums[i];
            }
            else{
                sd += nums[i];
            }
        }

        if(sd>dd || dd>sd){
            return true;
        }
        return false;
    }
}

File: 3515-find-if-digit-game-can-be-won/find-if-digit-game-can-be-won.py
class Solution:
    def canAliceWin(self, nums: List[int]) -> bool:
        sd =0
        dd = 0
        for i in range(0,len(nums)):
            if nums[i] > 9:
                dd += nums[i]
            else:
                sd += nums[i]
        if (sd > dd) or (dd > sd):
            return True
        return False